specs/BackgroundTask/BackgroundTaskBuilder.md

```diff
@@ -105,6 +105,7 @@ namespace Microsoft.Windows.ApplicationModel.Background
             );
 
         String Name{ set; get; };
+        Windows.ApplicationModel.Background.BackgroundTaskRegistrationGroup TaskGroup { set; get; };
```

Comment:
Do we need another round of API review for this, or the prior one is sufficient.

Reply:
This one was in spec earlier also, just implementation was missing